xhyve/xhyve.go

```diff
@@ -84,6 +85,7 @@ var (
 	ErrMachineNotExist = errors.New("machine does not exist")
 	diskRegexp         = regexp.MustCompile("^/dev/disk([0-9]+)")
 	kernelRegexp       = regexp.MustCompile(`(vmlinu[xz]|bzImage)[\d]*`)
+	kernelOptionRegexp = regexp.MustCompile(`\s*append`)
```

Comment:
It seems intended.
However, maybe there is `append` to `menu` is an edge case.
I leave this to you.